pisek/config/config_description.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, section: str) -> None:
 
     def similarity(self, section: str) -> int:
         if self.similarity_function is None:
-            return 5
+            return 5 * (self.section != section)
```

Comment:
Wouldn't this be clearer?

```suggestion
            return 5 * if self.section != section else 0
```